reference/library/src/morpheus-master/sass/modules/tutorial/_base.scss

```diff
@@ -0,0 +1,85 @@
+$alt-colour: #d36f00;
```

Comment:
Could you move this variable to https://github.com/sakaiproject/sakai/blob/master/reference/library/src/morpheus-master/sass/_defaults.scss?
Is good to have all variables in the same place to change them easily.
Thank !!